main.go

```diff
@@ -269,148 +374,62 @@ func keygenCmd(c *cli.Context) error {
 	return nil
 }
 
-// groupCmd reads the identity, check the threshold and outputs the group.toml
-// file
 func groupCmd(c *cli.Context) error {
```

Comment:
This group commands now fulfills two roles instead of one previously. We should add something in the readme about the process of "how to extend the group.toml file" stg like that. The process is stg like (1) get the current group.toml with the control port functionality and (2) call the `group` method to add new identities to it.

Reply:
The readme will be updated in #65 or after the PR is merged in the master to avoid rewriting it again and again

Reply:
Perfect.